docs/config_file.md

```diff
@@ -66,6 +66,7 @@ Chrome, Chrome Canary, Chromium, Firefox, IE, Opera, PhantomJS, Safari, Safari T
 
 ### Config-level options:
 
+    additionalBrowserSocketEvents[Object]  event names (key) to custom event handlers functions to be added to the browser socket
```

Comment:
Can we follow the existing naming conventions here? This would become `additional_browser_socket_events`. Additionally, the description is quite confusing - the type is `Object` but you say 'event names (key) to custom event handlers functions...'.

It maybe should be:

> an object containing keys corresponding to event names that point to handler functions, which are to be added to the browser socket